src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1259,7 +1259,7 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMProposeSet> const& m)
         return;
     }
 
-    bool isTrusted = app_.getUNL ().nodeInUNL (signerPublic);
+    auto const isTrusted = app_.validators().trusted (signerPublic);
```

Comment:
The type if `isTrusted` is `boost::optional<string>`, but the variable name signals that it's of type `bool` (and that's how it's used below). We could cast the result of `trusted` to `bool` (or add a helper to to validators that returns a bool).